docs/windows.md

```diff
@@ -65,28 +89,31 @@ the Windows Nodes in the demo.
 
 First install Antrea (v0.13.0+ is required for containerd).
 
+An example command that downloads and applies `antrea.yml` is like below:
+
 ```bash
-# Example:
-kubectl apply -f https://github.com/antrea-io/antrea/releases/download/v1.6.0/antrea.yml
+kubectl apply -f https://github.com/antrea-io/antrea/releases/download/v1.13.0/antrea.yml
```

Comment:
I saw some other examples use <TAG> to express the release version. Could you use the same way in all examples?

Reply:
Sure, updated.